### DISCUSSION TITLE
Can I use Issac ROS in the yocto?

### DISCUSSION THREAD

**Discussion by bchoineubility** [OE4T/meta-tegra#1520] [Q&A]
Hi,

Is there any guide to use Issac ROS in the yocto target?
BR,
Mark

**Comment by dwalkes** [comment]
Looks like you've already posted on the discussions at https://github.com/OE4T/tegra-demo-distro/issues/275 and https://github.com/OE4T/meta-tegra-community/issues/104 where others have referenced it.  I haven't used it myself so I won't be able to provide any more guidance beyond what I already shared [here](https://github.com/OE4T/tegra-demo-distro/issues/275#issuecomment-1704332953).

**Reply by bchoineubility** [reply]
@dwalkes thansk for you reply.

I will consider running ros2 humble container on the target built with yocto.

BR,
Mark
